.babelrc

```diff
@@ -1,6 +1,7 @@
 {
   "presets": [
     [ "es2015", { "modules": false } ],
+    "stage-0",
```

Comment:
`stage-0` gives you stages 1-3, plus these:

- transform-do-expressions
- transform-function-bind

I'm willing to bet you don't actually need `stage-0`... what syntax are you going for?